test/unit/spec/v2/view-builder/views/IdentifierView_spec.js

```diff
@@ -146,4 +150,62 @@ describe('v2/view-builder/views/IdentifierView', function() {
     expect(testContext.view.model.get('identifier')).toEqual('testUsername');
     expect(testContext.view.$el.find('.o-form-input-name-identifier input').val()).toEqual('testUsername');
   });  
+
+  it('view updates model and view correctly with rememberMe feature ON', function() {
```

Comment:
"fills in identifier form with username from cookie when rememberMe feature is enabled" ?